permissions/src/main/java/org/openmbee/sdvc/permissions/config/PermissionInit.java

```diff
@@ -0,0 +1,81 @@
+package org.openmbee.sdvc.permissions.config;
+
+import org.openmbee.sdvc.core.config.Roles;
+import org.openmbee.sdvc.core.config.Privileges;
+import org.openmbee.sdvc.data.domains.global.Group;
+import org.openmbee.sdvc.data.domains.global.Privilege;
+import org.openmbee.sdvc.data.domains.global.Role;
+import org.openmbee.sdvc.rdb.repositories.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+import javax.transaction.Transactional;
+import java.util.*;
+
+@Component
+@Transactional
+public class PermissionInit implements ApplicationListener<ApplicationReadyEvent> {
+    @Autowired
```

Comment:
instead of having autowired on a field, prefer to use it on a setter function (autowiring on private fields is discourage)